pages/common/img2pdf.md

```diff
@@ -0,0 +1,28 @@
+# img2pdf
+
+> Command-line lossless converter of raster images to PDF.
+> More information: <https://gitlab.mister-muffin.de/josch/img2pdf>.
+
+- Convert multiple images to PDF, each image being on its own page:
```

Comment:
```suggestion
- Convert multiple images to PDF, each image being on its own document:
```
```suggestion
- Convert multiple images to PDF, each image being on its own PDF:
```

Reply:
Actually, it ouputs them to a single file. This PDF contains multiples pages, one per input.

Reply:
:+1: